src/locales/ja/translation.json

```diff
@@ -73,16 +72,16 @@
   "Collapse": "畳む",
   "Expand": "展開",
   "Add Layer": "レイヤー追加",
-  "Zoom:": "ズーム:",
   "Search": "検索",
+  "Zoom:": "ズーム:",
   "Close popup": "ポップアップを閉じる",
   "cursor:": "カーソル",
   "center:": "中央:",
   "rotation:": "回転角度:",
   "Close modal": "モーダルを閉じる",
   "Debug": "デバッグ",
   "Options": "設定",
-  "<0>Open in OSM</0> &mdash; Opens the current view on openstreetmap.org": "現在のビューを <0>openstreetmap.org で開く</0>",
+  "<0>Open in OSM</0> &mdash; Opens the current view on openstreetmap.org": "__STRING_NOT_TRANSLATED__",
```

Comment:
Same here.
I think there's some bug related to the dash...

Reply:
Yeah, I tried a few variations and was unsuccessful looking for a solution. It does indeed seem related to the dash.

Reply:
<img width="416" alt="image" src="https://github.com/user-attachments/assets/db986a39-6f44-447c-a099-efda6242200c">

Screenshot after removing this line.